srtcore/core.cpp

```diff
@@ -8017,6 +8185,9 @@ void CUDT::sendLossReport(const std::vector<std::pair<int32_t, int32_t> > &loss_
 
 int CUDT::processData(CUnit *in_unit)
 {
+    if (m_bClosing)
+        return -1;
+
```

Comment:
The group code has given it more of a chance to receive a packet and process it while closing has started.